src/plugins/stake-plugins/thorchainSavers/tcSaversPlugin.ts

```diff
@@ -427,9 +451,24 @@ const stakeRequest = async (opts: EdgeGuiPluginOptions, request: ChangeQuoteRequ
         allocationType: 'fee',
         pluginId,
         currencyCode,
-        nativeAmount: toFixed(totalFee, 0, 0)
+        nativeAmount: toFixed(fee, 0, 0)
+      },
+      {
+        allocationType: 'stakeFee',
```

Comment:
Would `'slippageFee'` be a more appropriate name?

Reply:
sam ok for now

Reply:
renamed to `deductedFee`. renamed `fee` to `networkFee`